documentation/src/main/asciidoc/topics/ref_protostream_annotations.adoc

```diff
@@ -0,0 +1,152 @@
+[id='protostream-annotations_{context}']
+= ProtoStream annotations
+The ProtoStream API includes annotations that you can add to Java applications to define Protobuf schemas, which provide a structured format for your objects.
+
+This topic provides additional details about ProtoStream annotations.
+You should refer to the documentation in the annotations package for complete information.
```

Comment:
It would be could to state the fqn of the annotations package.

Reply:
Even better would be a link to some Javadocs that are available online.

Reply:
@oraNod I'd rather add one more link inline instead of pointing the user to the end of the topic for the link.